song_match/song_match.py

```diff
@@ -15,6 +15,8 @@ class SongMatch:
 
     def __init__(self):
         self._robot = None
+        self._tappedNoteIndex = 0
```

Comment:
We should use underscores instead of camel case.

Checkout the official [PEP 8](https://www.python.org/dev/peps/pep-0008/) Python style guide.

PyCharm is configured with PEP 8 and should underline those variables and suggest you use underscores instead.

Reply:
Ok cool. Thanks for the PEP 8 resource.